src/common/types.ts

```diff
@@ -43,9 +43,14 @@ export enum TestLevel {
 
 export interface ApiError extends Error {
   errorCode: string;
-  name: string;
 }
 
+export interface ApiPromoteResponse {
+  jobId: string;
+}
+
+export type ApiQuickResponse = ApiPromoteResponse;
```

Comment:
Why do we need a specific type for quick? It is calling the same api and receiving the same response than the "regular" promote.

Reply:
Ok! We can use the same type. I made it like this to differentiate between the two operations, but we can change this.